tests/conftest.py

```diff
@@ -103,3 +159,19 @@ def exec_context():
 @pytest.fixture(scope="session")
 def repo_root():
     return Path(os.path.abspath(__file__)).parents[1]
+
+
+@pytest.fixture(scope="session")
+def action_docker_image_id(repo_root):
+    "Build a docker image using local source and return the tag"
+    _, iidfile = tempfile.mkstemp()
+    try:
+        proc = subprocess.run(
+            ["docker", "buildx", "build", ".", "--load", "--iidfile", iidfile],
```

Comment:
Does this run in any CI tooling right now? If so, do we need to do anything special to make sure it can interact with Docker? I know on Buildkite, because our builds are already running inside a container, we need to mount the docker socket as a volume.

Reply:
There is no CI set up for this. For not this just makes it possible to test locally.

I do want to get this into a CI. Not sure where yet. Even our action docker image should be built and published through CI, if not already.